mint/sources/mint.move

```diff
@@ -31,6 +31,7 @@ const ERROR_INVALID_TOKEN_ID: u64 = 7;
 const ERROR_PREMINT_NOT_COMPLETED: u64 = 8;
 const ERROR_NO_MYTHICS_AVAILABLE: u64 = 9;
 const ERROR_INVALID_RANGE: u64 = 10;
+const EWrongAuctionContract: u64 = 11;
```

Comment:
```suggestion
const ERROR_WRONG_AUCTION_CONTRACT: u64 = 11;
```

Reply:
maybe lets rename to auction `id` instead since it an object, not a contract

Reply:
Let's do other PR to fix the error naming..